harper-core/src/linting/lint_group.rs

```diff
@@ -187,7 +188,8 @@ create_lint_group_config!(
     PluralConjugate => false,
     OxfordComma => true,
     PronounContraction => true,
-    CurrencyPlacement => true
+    CurrencyPlacement => true,
+    LetUsRedundancy => true
 );
```

Comment:
Does Rust support tge following notation as other languages do.


```suggestion
    CurrencyPlacement => true,
    LetUsRedundancy => true,
);
```

It could avoid updating unrelated line on each rule addition

Reply:
Rust does, but this is actually on of our macros. I don't think it would be worth the effort to add that in.

Reply:
turns out it's really easy

add `$(,)?` to the end of the macro signature

```rust
    ($($linter:ident => $default:expr),* $(,)?) => {
```